clconf/in_memory_template.go

```diff
@@ -91,7 +93,10 @@ func NewTemplate(name, text string, config *TemplateConfig) (*Template, error) {
 		addCryptFuncs(funcMap, config.SecretAgent)
 	}
 
-	tmpl, err := template.New(name).Funcs(funcMap).Parse(text)
+	tmpl, err := template.New(name).
```

Comment:
wrap consistently makes it easier to follow:

```suggestion
	tmpl, err := template.
		New(name).
```